data/yaml/venues/teachnlp.yaml

```diff
@@ -0,0 +1,2 @@
+acronym: TeachNLP
+name: The Sixth Workshop on Teaching NLP
```

Comment:
This is the sixth meeting, so it seems like it probably exists? Digging around I found this: https://aclanthology.org/venues/teachingnlp/